packages/@ourworldindata/grapher/src/core/GrapherInterface.ts

```diff
@@ -149,6 +149,7 @@ export const grapherKeysToSerialize = [
     "hideRelativeToggle",
     "entityType",
     "entityTypePlural",
+    "yVariableType",
```

Comment:
I'm not a fan of `yVariableType `- this makes me think of a data type or so but this is actually a label. What about `facettingLabelByYVariables` or similar?

Reply:
Yeah.. I'm not a fan of `yVariableType` either... I struggled to come up with something and then landed on the one that was closest to `entityType`.

I tried to choose a name that describes what it is rather than what it is used for in the UI (in case we wanted to use it for something else in the future). But I don't have a particular second use case in mind, so we might as well give it a use case-specific name.

I like `facettingLabelByYVariables`! How about `facetControlLabelForSplitByVariable`?